docs/webService.md

```diff
@@ -111,4 +111,14 @@ public @interface URL {
 ```
 You can then use your custom constraint validator such as the [built in constraints](https://beanvalidation.org/2.0/spec/#builtinconstraints) by using `@URL`.
 
+Several annotations don't work on `List`. For example, `@Email` only works on `String`. You can create your own validator as showed above or use type level annotations. In Bean Validation (2.0), all provided constraints have `ElementType.TYPE_USE`.
```

Comment:
Can you be more precise for this sentence ? I don't understand everything. What is the purpose of ElementType.TYPE_USE  ?
In Bean Validation (2.0), all provided constraints have `ElementType.TYPE_USE`.

Reply:
From documentations : https://docs.oracle.com/javase/8/docs/api/java/lang/annotation/ElementType.html
The TYPE_USE constant includes type declarations and type parameter declarations as a convenience for designers of type checkers which give semantics to annotation types. For example, if the annotation type NonNull is meta-annotated with @Target(ElementType.TYPE_USE), then @NonNull class C {...} could be treated by a type checker as indicating that all variables of class C are non-null, while still allowing variables of other classes to be non-null or not non-null based on whether @NonNull appears at the variable's declaration.